documentation/Experian Commerce Cloud Integration Manual.md

```diff
@@ -246,12 +246,6 @@ To check the imported site preferences go to **Administration > Site Development
 ![SGJC EDQ Unicorn 4](https://raw.githubusercontent.com/JoseCastilloExperian/edqCommerceCloud/master/EDQ%20Cartridge%20Manual%20imgs/321EDQUnicorn4.png)
 
 ####  3.2.2\. Storefront Reference Architecture
-**EDQHeaders integration**
-1.	Go to int_experian_sfra cartridge and open the template located in **templates > default > common > htmlHead.isml.**
-2. The `<isinclude template="EDQ/EDQHeaders" />` line has been added at the end of **“htmlHead.isml”** file.
-
-![SFRA EDQ Headers](https://raw.githubusercontent.com/JoseCastilloExperian/edqCommerceCloud/master/EDQ%20Cartridge%20Manual%20imgs/322EDQHeaders.png)
-
```

Comment:
I assume this is no longer necessary? Is this documented elsewhere or is it outdated?

Reply:
Is no longer necessary since it was removed due the SFRA v4.4 compability.